fieldservice_account/models/fsm_order.py

```diff
@@ -46,6 +46,7 @@ def _compute_employee(self):
     def _compute_total_cost(self):
         for order in self:
             order.total_cost = 0.0
+            rate = 0
```

Comment:
The rate is added and there is no error message. But still, because of this
```
for emp in line.user_id.employee_ids
```
If the line with selected employee_id that not have user_id, the total_cost will still be 0.0 (even employee have timesheet rates)

I am not sure this is what is intended? But for me, when seeing with my eye it not make sense.

Reply:
@kittiu I agree here that this method seems unnecessary, we can just get the timesheet_cost from the employee_id on the line. Made changes in my last commit.